README.md

```diff
@@ -34,28 +40,24 @@ uaac client add <your-client-id> \
 - Unable to create an account still? Troubleshoot [here](https://docs.cloudfoundry.org/adminguide/uaa-user-management.html#creating-admin-users)
 
 ### Set the environment variables
-If you are testing locally, export these variables. If you are deploying to cloud foundry, modify the manifest.yml
+If you are testing locally, export these variables. If you are deploying to cloud foundry, modify the `manifest.yml`
+- `GOPATH`: The absolute path to your project root. If you followed the cloning instructions above, this path should end with `cg-deck-ws`
 - `CONSOLE_CLIENT_ID`: Registered client id with UAA.
 - `CONSOLE_CLIENT_SECRET`: The client secret.
 - `CONSOLE_HOSTNAME`: The URL of the service itself.
 - `CONSOLE_LOGIN_URL`: The base URL of the auth service. i.e. `https://login.domain.com`
 - `CONSOLE_UAA_URL`: The URL of the UAA service. i.e. `https://uaa.domain.com`
 - `CONSOLE_API_URL`: The URL of the API service. i.e. `http://api.domain.com`
 - `CONSOLE_LOG_URL`: The URL of the loggregator service. i.e. `http://loggregator.domain.com`
-- `PPROF_ENABLED`: An optional variable. If set to `true` or `1`, will turn on `/debug/pprof` endpoints as seen [here](https://golang.org/pkg/net/http/pprof/)
-- `CG_STYLE_PATH`: The absolute path to your `cg-style` repo. Used for webpack in development
+- `PPROF_ENABLED`: <optional> If set to `true` or `1`, will turn on `/debug/pprof` endpoints as seen [here](https://golang.org/pkg/net/http/pprof/)
+- `CG_STYLE_PATH`: <optional> The absolute path to your `cg-style` repo. If set, will use a local copy of `cloudgov-style` to build the front end application.
 
 ## Front end
-Install front end dependencies, including the `cloudgov-style` library
+Install front end dependencies:
```

Comment:
Add note about where in the dir structure to run this cmd from. e.g. `/path/to/folder/cg-deck-ws/src/github.com/18F/cg-deck`